ibm/service/power/data_source_ibm_pi_volumes.go

```diff
@@ -0,0 +1,255 @@
+							Type:        schema.TypeString,
+						},
+						Attr_IOThrottleRate: {
+							Computed:    true,
+							Description: "Amount of iops assigned to the volume",
+							Type:        schema.TypeString,
+						},
+						Attr_LastUpdateDate: {
+							Computed:    true,
+							Description: "The last updated date of the volume.",
+							Type:        schema.TypeString,
+						},
+						Attr_MasterVolumeName: {
+							Computed:    true,
+							Description: "The master volume name.",
+							Type:        schema.TypeString,
+						},
+						Attr_MirroringState: {
+							Computed:    true,
+							Description: "Mirroring state for replication enabled volume.",
+							Type:        schema.TypeString,
+						},
+						Attr_OutOfBandDeleted: {
+							Computed:    true,
+							Description: "Indicates if the volume does not exist on storage controller.",
+							Type:        schema.TypeBool,
+						},
+						Attr_PrimaryRole: {
+							Computed:    true,
+							Description: "Indicates whether master/auxiliary volume is playing the primary role.",
+							Type:        schema.TypeString,
+						},
+						Attr_ReplicationEnabled: {
+							Computed:    true,
+							Description: "Indicates if the volume should be replication enabled or not.",
+							Type:        schema.TypeBool,
+						},
+						Attr_ReplicationSites: {
+							Computed:    true,
+							Description: "List of replication sites for volume replication.",
+							Elem:        &schema.Schema{Type: schema.TypeString},
+							Type:        schema.TypeList,
+						},
+						Attr_ReplicationStatus: {
+							Computed:    true,
+							Description: "The replication status of the volume.",
+							Type:        schema.TypeString,
+						},
+						Attr_ReplicationType: {
+							Computed:    true,
+							Description: "The replication type of the volume, metro or global.",
+							Type:        schema.TypeString,
+						},
+						Attr_Shareable: {
+							Computed:    true,
+							Description: "Indicates if the volume is shareable between VMs.",
+							Type:        schema.TypeBool,
+						},
+						Attr_Size: {
+							Computed:    true,
+							Description: "The size of the volume in GB.",
+							Type:        schema.TypeInt,
+						},
+						Attr_State: {
+							Computed:    true,
+							Description: "The state of the volume.",
+							Type:        schema.TypeString,
+						},
+						Attr_UserTags: {
+							Computed:    true,
+							Description: "List of user tags attached to the resource.",
+							Elem:        &schema.Schema{Type: schema.TypeString},
+							Set:         schema.HashString,
+							Type:        schema.TypeSet,
+						},
+						Attr_VolumeName: {
+							Computed:    true,
+							Description: "The name of the volume.",
+							Type:        schema.TypeString,
+						},
+						Attr_VolumePool: {
+							Computed:    true,
+							Description: "Volume pool, name of storage pool where the volume is located.",
+							Type:        schema.TypeString,
+						},
```

Comment:
Can we add `VolumeType`?

Reply:
Yes it seems like it's missing from ibm_pi_volume data source as well